src/shipit_signoff/shipit_signoff/controllers/default_controller.py

```diff
@@ -0,0 +1,21 @@
+
+def create_step(uid) -> str:
```

Comment:
I like the idea of keeping the step API endpoints separate from the signoff-specific ones, but it feels a bit weird that they live so far apart. At the very least, I think they should be in the same directory.